src/schoology-extractor/schoology_extractor/api/request_client.py

```diff
@@ -5,14 +5,16 @@
 
 import time
 import random
+from typing import List, TypeVar
 
-from requests_oauthlib import OAuth1Session
+from requests_oauthlib import OAuth1Session  # type: ignore
 
 from .paginated_result import PaginatedResult
 
 
 DEFAULT_URL = "https://api.schoology.com/v1/"
 DEFAULT_PAGE_SIZE = 20
+T = TypeVar('T')
```

Comment:
I did not end up using that. Surprised that the checkers did not complain!

Reply:
Should I remove it?

Reply:
Yes, please do.